lib/libmbedtls/core/tee_lmd_provider.c

```diff
@@ -2629,7 +2683,31 @@ TEE_Result crypto_mac_init(void *ctx, uint32_t algo, const uint8_t *key,
 	case TEE_ALG_DES_CBC_MAC_PKCS5:
 	case TEE_ALG_DES3_CBC_MAC_NOPAD:
 	case TEE_ALG_DES3_CBC_MAC_PKCS5:
-		return TEE_ERROR_NOT_SUPPORTED;
+		cbc = (struct cbc_state *)ctx;
+
+		res = tee_algo_to_ltc_cipherindex(algo, &ltc_cipherindex);
```

Comment:
How come the mbedtls implementation of AES/DES/DES3 isn't used instead?

Reply:
Sorry not notice it before. I will refine it by using mbedtls.